benchmarkapps/BenchmarkServer/signalr.json

```diff
@@ -20,5 +20,8 @@
   "SignalREchoAll": {
     "ClientProperties": { "Scenario": "echoAll" },
     "Warmup": 0
+  },
+  "SignalREchoIdle": {
+    "ClientProperties": { "Scenario": "echoIdle"}
```

Comment:
nit: Space after `}`

Reply:
Do we want to set any other defaults for the idle scenario?

Reply:
I figured we could just keep the defaults for now maybe with the exception of the duration.  The built in default send period is every 10 minutes for the idle scenario.